.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+#### What's this PR do?
+
+#### Why are we doing this? How does it help us?
+
+#### How should this be manually tested?
+
+#### How should this change be communicated to end users?
+
+#### Are there any smells or added technical debt to note?
+
+#### What are the relevant tickets?
+
+#### Have you done the following, if applicable:
+***(optional: add explanation between parentheses)***
+
+* [ ] Added automated tests? *( )*
+* [ ] Added migrations? *( )*
```

Comment:
migrations might not be as relevant in this repo, but glad to have a PR template here!